deploy.sh

```diff
@@ -147,7 +147,6 @@ function _deploy() {
   # Apply image specific config
   kubectl set image -f config/kubernetes/${environment}/deployment.yaml \
           webapp=${docker_image_tag} \
-          metrics=${docker_image_tag} \
```

Comment:
Sorry, missed this one. This needs to be kept as it is used for the prometheus stuff.